test/ObjectField_test.js

```diff
@@ -50,7 +50,10 @@ describe("ObjectField", () => {
     });
 
     it("should render a customized title", () => {
-      const CustomTitleField = ({title}) => <div id="custom">{title}</div>;
+      const CustomTitleField = ({title, required}) => {
+        const legend = required ? title + '*' : title;
+        return <div id="custom">{legend}</div>;
+      };
```

Comment:
I think tests as the spec, so I wanted to update the example here to reflect the changes in the TitleField, even though the change does not affect the assertion.

`expect(node.querySelector("fieldset > #custom").textContent)`